src/pdf/DriverOverview/DriverOverviewPdfButton.tsx

```diff
@@ -67,6 +95,19 @@ const getParcelsForDelivery = async (parcelIds: string[]): Promise<ParcelsForDel
         dataWithNonNullClients.push({ ...parcel, client: parcel.client });
     }
 
+    const eventResponse = await getParcelEvents(parcelIds);
+    if (eventResponse.error || !eventResponse.data) {
+        return eventResponse;
+    }
+
+    // Find most recent label download event and store the number of labels (the event_data)
```

Comment:
we don't really need this comment, I think the code should aim to be self-explanatory where possible, for readability